subsys/bluetooth/controller/hci_internal.c

```diff
@@ -695,6 +695,7 @@ void hci_internal_le_supported_features(
 	features->params.extended_reject_indication = 1;
 	features->params.slave_initiated_features_exchange = 1;
 	features->params.le_ping = 1;
+	features->params.le_Power_class_1 = 1;
```

Comment:
Is it international that this is capital letter `P`?

Reply:
Yeah that is how it is in the struct and the .yml we use to generate it - for some reason